packages/react-core/src/components/Modal/Modal.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import ModalContent from './ModalContent';
+
+const propTypes = {
+  /** content rendered inside the Modal. */
+  children: PropTypes.node.isRequired,
+  /** additional classes added to the button */
+  className: PropTypes.string,
+  /** Flag to show the modal */
+  show: PropTypes.bool,
```

Comment:
lets make this `isOpen` to match the other boolean syntax of prefixing with `is` or `has`